src/scenes/AcceleratorRouter/Scoring/PhaseScores/LastUpdated.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useMemo } from 'react';
+
+import { Typography } from '@mui/material';
+
+import { useLocalization } from 'src/providers';
+import strings from 'src/strings';
+import { PhaseScores, Score } from 'src/types/Score';
+import { getLongDate } from 'src/utils/dateFormatter';
+
+interface LastUpdatedProps {
+  phaseScores: PhaseScores;
+}
+
+const LastUpdated = ({ phaseScores }: LastUpdatedProps) => {
+  const { activeLocale } = useLocalization();
+
+  const modifiedTime = useMemo(
+    () =>
+      phaseScores?.scores
+        .map((score: Score) => score.modifiedTime)
+        .sort()
+        .shift(),
+    [phaseScores]
+  );
+
+  if (!(activeLocale && modifiedTime)) {
```

Comment:
Try removing the activeLocale dependency and updating the language.. in case we can avoid activeLocale dependency in the component?

Reply:
It is here is because it is required by the `getLongDate` function. I wanted to remove it but in this case it is needed